src/displayapp/lv_pinetime_theme.c

```diff
@@ -215,8 +215,8 @@ static void basic_init(void) {
   lv_style_set_transition_prop_6(&style_cb_bg, LV_STATE_DEFAULT, LV_STYLE_OUTLINE_OPA);
 
   lv_style_reset(&style_cb_bullet);
-  lv_style_set_outline_opa(&style_cb_bullet, LV_STATE_FOCUSED, LV_OPA_TRANSP);
   lv_style_set_radius(&style_cb_bullet, LV_STATE_DEFAULT, LV_DPX(4));
+  lv_style_set_pattern_image(&style_cb_bullet, LV_STATE_CHECKED, LV_SYMBOL_OK);
```

Comment:
is `LV_SYMBOL_OK` defined by lvgl? Other times when we add a new glyph it needs to be defined in multiple places. Here it only is added to `fonts.json` and can be directly used?

Reply:
Yes, it's defined by LVGL. In this case we couldn't include Symbols.h anyway because it's a C++ file and this is a C file, unless #1258 was merged.

Reply:
ah thanks for the clarification and extra info!